### DISCUSSION TITLE
next-swc: plugin_modularize_imports add more functionalities

### DISCUSSION THREAD

**Discussion by JSerFeng** [vercel/next.js#41327] [Ideas]
### Describe the feature you'd like to request

Hi, is there some plan on improving the extensibility for plugin_modularize_imports of package next-swc ? Want some more abilities of plugin_modularize_imports, for example: add kebab case. like `{{kebabCase member}}`

### Describe the solution you'd like

Maybe pub an template register instance so we can use that to register our own functions.

### Describe alternatives you've considered

Or just add more naming conversion by default, kebabCase snakeCase etc

**Comment by balazsorban44** [comment]
Hi, this should already be supported as per the docs: https://nextjs.org/docs/advanced-features/compiler#modularize-imports. PR was here: https://github.com/vercel/next.js/pull/38583

**Reply by JSerFeng** [reply]
Thanks !
